service_geo_location/app/services/GeoLocationService.scala

```diff
@@ -1,45 +1,34 @@
-/*
- * MUSIT is a museum database to archive natural and cultural history data.
- * Copyright (C) 2016  MUSIT Norway, part of www.uio.no (University of Oslo)
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License,
- * or any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License along
- * with this program; if not, write to the Free Software Foundation, Inc.,
- * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
- */
-
 package services
 
 import com.google.inject.Inject
 import models.{Address, GeoNorwayAddress}
-import play.api.{Configuration, Logger}
+import net.ceedubs.ficus.Ficus._
+import net.ceedubs.ficus.readers.ArbitraryTypeReader._
+import no.uio.musit.ws.ViaProxy
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
 import play.api.libs.json._
 import play.api.libs.ws.WSClient
-import GeoLocationService._
-import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.{Configuration, Logger}
 
 import scala.concurrent.Future
 
-class GeoLocationService @Inject()(config: Configuration, ws: WSClient) {
+case class GeoLocationConfig(url: String, hitsPerResult: Int)
+
+class GeoLocationService @Inject()(ws: WSClient)(implicit config: Configuration)
+    extends ViaProxy {
```

Comment:
See above comment about creating an object that extends the  `ViaProxy` trait to allow the implicit scope to be imported instead of extending the class.